site/_posts/2020-06-29-django-and-oauth.md

```diff
@@ -13,43 +13,43 @@ In this tutorial, we'll build a basic Django web application which does user reg
 
 <!--more-->
 
+TODO do we want to use this?
 {% include _image.liquid src="/assets/img/blogs/social-sign-in-django/headerimage.png" alt="Header Image" class="img-fluid" figure=false %}
 
-The application itself will be very simple: it will let users put in their birthdays and keep this “secret" information safe for them, but with these basics in place, you'll see how FusionAuth works and can extend the application to do whatever you need. You can, as always, [skip ahead and view the code](https://github.com/fusionauth/fusionauth-example-python-django).
+The application itself will be very simple: it will let users put in their birthdays and keep this "secret" information safe for them. With these basics in place, you'll see how FusionAuth works and can extend the application to do whatever you need. You can, as always, [skip ahead and view the code](https://github.com/fusionauth/fusionauth-example-python-django).
```

Comment:
The application itself is very simple: it will let users put in their birthdays and keep this "secret" information safe for them. With these basics in place, you'll see how FusionAuth works and how it can extend the application to do whatever you need.